osbenchmark/workload/params.py

```diff
@@ -1072,11 +1072,8 @@ def __init__(self, workloads, params, query_params, **kwargs):
             neighbors_corpora = self.extract_corpora(self.neighbors_data_set_corpus, self.neighbors_data_set_format)
             self.corpora.extend(corpora for corpora in neighbors_corpora if corpora not in self.corpora)
 
-    def _validate_neighbors_data_set(self):
-        if not self.data_set_path and not self.data_set_corpus:
-            raise exceptions.ConfigurationError(
-                "Dataset is missing. Provide either dataset file path or valid corpus.")
-        if self.data_set_path and self.data_set_corpus:
+    def _validate_neighbors_data_set(self, file_path, corpus):
+        if file_path and corpus:
```

Comment:
Since `self.data_set_path` and `self.data_set_corpus` have been replaced with `file_path` and `corpus`, should the exception be updated to reflect those new arguments?

Reply:
Ack